decidim-core/spec/system/notifications_spec.rb

```diff
@@ -112,6 +112,26 @@
     end
   end
 
+  context "with comment notifications" do
```

Comment:
Can we move this spec to `decidim-comments/spec/services/decidim/comments/new_comment_notification_creator_spec.rb` ?

Reply:
done 👌